compiler/resolution/functionResolution.cpp

```diff
@@ -11482,6 +11483,59 @@ static void checkSpeciallyNamedMethods() {
   }
 }
 
+static void applyGpuAttributesToIterableExprs() {
+  for_alive_in_Vec(BlockStmt, block, gBlockStmts) {
```

Comment:
I am slightly worried about a compilation time hit from this. Not to do anything at this point, but just something to keep an eye on.